src/clj/game/cards/resources.clj

```diff
@@ -495,13 +501,14 @@
                   :msg "gain 1 [Credits]"
                   :req (req (:runner-phase-12 @state))
                   :counter-cost [:credit 1]
+                  :async true
                   :effect (req (gain-credits state :runner 1)
-                               (when (zero? (get-counters (get-card state card) :credit))
-                                 (trash state :runner card {:unpreventable true})
-                                 (draw state :runner 1)
-                                 (system-msg state :runner (str "trashes Crowdfunding"
-                                                               (when (not (empty? (:deck runner)))
-                                                                " and draws 1 card")))))}
+                               (if (zero? (get-counters (get-card state card) :credit))
```

Comment:
Any reason to change from `when` to `if` here?

Reply:
Amusingly, yes, as the base ability is `:async` i made it `if` to have an else branch marking the effect as completed. I just forgot to actually write that branch.